.gitmodules

```diff
@@ -1,12 +1,9 @@
-[submodule "public/custom/ReactionPublicCustomFolder"]
-	path = public/custom/ReactionPublicCustomFolder
-	url = git@github.com:evereveofficial/ReactionPublicCustomFolder.git
 [submodule "imports/plugins/custom"]
 	path = imports/plugins/custom
 	url = git@github.com:evereveofficial/eePlugins.git
-# [submodule "private/settings"]
```

Comment:
they're still there, just we had one set that was commented out for some reason?